21/10/2020 08:36:52 PM Python Script is: morePVs.py
21/10/2020 08:36:52 PM study_name = pt_mike
21/10/2020 08:36:54 PM Running Scenario number 0 
21/10/2020 08:36:55 PM Completed Scenario 0
21/10/2020 08:36:55 PM Running Scenario number 1 
21/10/2020 08:36:57 PM 


 Exception !!!!!!
Traceback (most recent call last):
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 2640, in main
    runScenario(s)
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 2559, in runScenario
    scenario = Scenario(scenario_name=scenario_name)
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 1875, in __init__
    exit(msg)
  File "/Users/mikeroberts/anaconda3/lib/python3.7/_sitebuiltins.py", line 26, in __call__
    raise SystemExit(code)
SystemExit: ******Exception: Tariff EN_PT is not in tariff_lookup.csv
